collectors/collectors.go

```diff
@@ -19,11 +19,16 @@ package collectors
 import (
 	"time"
 
+	"regexp"
+
 	"github.com/prometheus/client_golang/prometheus"
+	"k8s.io/api/core/v1"
 )
 
 var (
-	resyncPeriod = 5 * time.Minute
+	invalidLabelCharRE = regexp.MustCompile(`[^a-zA-Z0-9_]`)
+
+	ResyncPeriod = 5 * time.Minute
```

Comment:
Could we make `ResyncPeriod ` to configurable?

Reply:
If that is necessary, we can do this in another PR instead of this one. :)